"dotnet build" complains about missing nuget packages

### Description

I tried to build maui main branch with `dotnet cake --target=VS --pack` and tried with the opening Visual Studio in version 17.4.4 to build an already existing maui application and end up in this situation like here https://github.com/dotnet/maui/issues/3923. I am right now not sure if manual install the workload fixed it (BTW
the weird part is that also if you remove the android target from csproj and remove the android platform folder it still requires it). Anyway I end up now having issue that basic nuget packages could not be found:

VS log:
```
1>------ Erstellen gestartet: Projekt: MauiApp3, Konfiguration: Debug Any CPU ------
Fehler beim Wiederherstellen des NuGet-Pakets. Detaillierte Warnungen und Fehler finden Sie im Fenster mit der Fehlerliste.
Fehler beim Wiederherstellen von "C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj" (in "2,28 sec").
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.AspNetCore.App.Runtime.win-x64" der Version (= 7.0.0-rtm.22512.1) wurde nicht gefunden.
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 110 Version(en) gefunden in "nuget.org" [ Nächste Version: 7.0.0 ]
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\trivalik\source\repos\maui\bin\dotnet\library-packs.
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.NETCore.App.Runtime.win-x64" der Version (= 7.0.0-rtm.22511.4) wurde nicht gefunden.
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 109 Version(en) gefunden in "nuget.org" [ Nächste Version: 7.0.0 ]
1>C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\trivalik\source\repos\maui\bin\dotnet\library-packs.
1>Die Erstellung des Projekts "MauiApp3.csproj" ist abgeschlossen -- FEHLER.
```

dotnet build log (with set environment variables https://github.com/dotnet/maui/blob/cb2014b6ffb8e097be1b5301f0a9a7cace11f213/eng/cake/helpers.cake#L95) :
```
C:\Users\trivalik\source\repos\MauiApp3>dotnet build
MSBuild version 17.5.0-preview-22507-04+9e6f1455a for .NET
  Wiederherzustellende Projekte werden ermittelt...
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.NETCore.App.Runtime.w
in-x64" der Version (= 7.0.0-rtm.22511.4) wurde nicht gefunden. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 109 Version(en) gefunden in "nuget.org
" [ Nächste Version: 7.0.0 ] [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\rob
o\source\repos\maui\bin\dotnet\library-packs. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.AspNetCore.App.Runtim
e.win-x64" der Version (= 7.0.0-rtm.22512.1) wurde nicht gefunden. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 110 Version(en) gefunden in "nuget.org
" [ Nächste Version: 7.0.0 ] [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\rob
o\source\repos\maui\bin\dotnet\library-packs. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
  Fehler beim Wiederherstellen von "C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj" (in "2,21 sec").

Fehler beim Buildvorgang.

C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.NETCore.App.Runtime.w
in-x64" der Version (= 7.0.0-rtm.22511.4) wurde nicht gefunden. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 109 Version(en) gefunden in "nuget.org
" [ Nächste Version: 7.0.0 ] [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\rob
o\source\repos\maui\bin\dotnet\library-packs. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102: Das Paket "Microsoft.AspNetCore.App.Runtim
e.win-x64" der Version (= 7.0.0-rtm.22512.1) wurde nicht gefunden. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 110 Version(en) gefunden in "nuget.org
" [ Nächste Version: 7.0.0 ] [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
C:\Users\trivalik\source\repos\MauiApp3\MauiApp3\MauiApp3.csproj : error NU1102:   - 0 Version(en) gefunden in C:\Users\rob
o\source\repos\maui\bin\dotnet\library-packs. [C:\Users\trivalik\source\repos\MauiApp3\MauiApp3.sln]
    0 Warnung(en)
    2 Fehler

Verstrichene Zeit 00:00:05.64

Workloadupdates sind verfügbar. Führen Sie "dotnet workload list" aus, um weitere Informationen zu erhalten.

C:\Users\trivalik\source\repos\MauiApp3>dotnet workload list

Installierte Workload-ID      Manifestversion          Installationsquelle
-------------------------------------------------------------------------------------
android                       33.0.4/7.0.100           SDK 7.0.100, VS 17.4.33213.308
ios                           16.2.1004/7.0.100        SDK 7.0.100, VS 17.4.33213.308
maccatalyst                   16.2.1004/7.0.100        SDK 7.0.100, VS 17.4.33213.308
macos                         13.1.1004/7.0.100        SDK 7.0.100
tizen                         7.0.100/7.0.100          SDK 7.0.100
tvos                          16.1.1501/7.0.100        SDK 7.0.100
maui-windows                  7.0.100-dev/7.0.100      VS 17.4.33213.308
maui-maccatalyst              7.0.100-dev/7.0.100      VS 17.4.33213.308
maui-ios                      7.0.100-dev/7.0.100      VS 17.4.33213.308
maui-android                  7.0.100-dev/7.0.100      VS 17.4.33213.308

Verwenden Sie "dotnet workload search", um zusätzliche zu installierende Workloads zu finden.

Updates sind für die folgenden Workloads verfügbar: android ios maccatalyst macos tizen tvos. Führen Sie "dotnet workload update" aus, um die neueste Updates zu erhalten.


C:\Users\trivalik\source\repos\MauiApp3>where dotnet
C:\Users\trivalik\source\repos\maui\bin\dotnet\dotnet.exe
C:\Program Files\dotnet\dotnet.exe
```

### Steps to Reproduce

1. get main branch (41553e16dcc51ce3dd97b20a7503058170848bcb)
2. open cmd and start `dotnet cake --target=VS --pack` (restore seems not necessary after once done?)
3. wait until VS opened and close the Micorosft.Maui solution (stucks some times and VS reports sometimes some exceptions about loading the project), on final test I had no such VS exceptions or any other trouble, so that this is not the root cause
4. create new maui app and press "Windows Machine" to run --> end up in missing nuget packages


To test manually with `dotnet` at cmd I used to prepare:

set PATH=C:/Users/trivalik/source/repos/maui/bin/dotnet;%PATH%
set DOTNET_INSTALL_DIR=C:/Users/trivalik/source/repos/maui/bin/dotnet
set DOTNET_MSBUILD_SDK_RESOLVER_CLI_DIR=C:/Users/trivalik/source/repos/maui/bin/dotnet
set DOTNET_MULTILEVEL_LOOKUP=0
set DOTNET_ROOT=C:/Users/trivalik/source/repos/maui/bin/dotnet
set MSBuildEnableWorkloadResolver=true

### Link to public reproduction project repository

https://github.com/dotnet/maui/

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows, I was *not* able test on other platforms

### Affected platform versions

Windows SDK 19041 and 20348 (build number manually changed in csproj)

### Did you find any workaround?

No.

### Relevant log output

_No response_

--- a/eng/Version.Details.xml
+++ b/eng/Version.Details.xml
@@ -1,41 +1,5 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.Dotnet.Sdk.Internal" Version="7.0.100-rtm.22519.39" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
-      <Uri>https://github.com/dotnet/installer</Uri>
-      <Sha>e6dd91c290b808f971a1ac69c2fb29395bbf1051</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="7.0.0-rtm.22511.4" CoherentParentDependency="Microsoft.Android.Sdk.Windows">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>d25158d0dffd699340cedcfd43324c87a809a214</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.Android.Sdk.Windows" Version="33.0.4">
-      <Uri>https://github.com/xamarin/xamarin-android</Uri>
-      <Sha>8f1d9a47205ead80132661f68b0cee9ed0e0220b</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.MacCatalyst.Sdk" Version="16.2.1026">
-      <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>9ffa71bdb3432581640b5830c775c9bfd0c16902</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.macOS.Sdk" Version="13.1.1026">
-      <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>9ffa71bdb3432581640b5830c775c9bfd0c16902</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.iOS.Sdk" Version="16.2.1026">
-      <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>9ffa71bdb3432581640b5830c775c9bfd0c16902</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.tvOS.Sdk" Version="16.1.1523">
-      <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>9ffa71bdb3432581640b5830c775c9bfd0c16902</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100" Version="7.0.0-rtm.22504.3" CoherentParentDependency="Microsoft.NETCore.App.Ref">
-      <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>daca2015ce74956591df9c9dc7ee732af7863b42</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.NET.Workload.Emscripten.net7.Manifest-7.0.100" Version="7.0.0-rtm.22504.3" CoherentParentDependency="Microsoft.NETCore.App.Ref">
-      <Uri>https://github.com/dotnet/emsdk</Uri>
-      <Sha>daca2015ce74956591df9c9dc7ee732af7863b42</Sha>
-    </Dependency>
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -3,15 +3,15 @@
     <!-- Current previous .NET SDK major version's stable release of MAUI packages -->
     <MicrosoftMauiPreviousDotNetReleasedVersion>6.0.547</MicrosoftMauiPreviousDotNetReleasedVersion>
     <!-- dotnet/installer -->
-    <MicrosoftDotnetSdkInternalPackageVersion>7.0.100-rtm.22519.39</MicrosoftDotnetSdkInternalPackageVersion>
+    <MicrosoftDotnetSdkInternalPackageVersion>7.0.102</MicrosoftDotnetSdkInternalPackageVersion>
     <!-- dotnet/runtime -->
-    <MicrosoftNETCoreAppRefPackageVersion>7.0.0-rtm.22511.4</MicrosoftNETCoreAppRefPackageVersion>
+    <MicrosoftNETCoreAppRefPackageVersion>7.0.2</MicrosoftNETCoreAppRefPackageVersion>
     <!-- NOTE: should eventually revert back to $(MicrosoftNETCoreAppRefPackageVersion) in .NET 7 -->
     <MicrosoftExtensionsPackageVersion>7.0.0</MicrosoftExtensionsPackageVersion>
     <MicrosoftExtensionsServicingPackageVersion>7.0.0</MicrosoftExtensionsServicingPackageVersion>
     <SystemCodeDomPackageVersion>7.0.0</SystemCodeDomPackageVersion>
     <!-- xamarin/xamarin-android -->
-    <MicrosoftAndroidSdkWindowsPackageVersion>33.0.4</MicrosoftAndroidSdkWindowsPackageVersion>
+    <MicrosoftAndroidSdkWindowsPackageVersion>33.0.26</MicrosoftAndroidSdkWindowsPackageVersion>
     <!-- xamarin/xamarin-macios -->
     <MicrosoftMacCatalystSdkPackageVersion>16.2.1026</MicrosoftMacCatalystSdkPackageVersion>
     <MicrosoftmacOSSdkPackageVersion>13.1.1026</MicrosoftmacOSSdkPackageVersion>
@@ -20,8 +20,7 @@
     <!-- Samsung/Tizen.NET -->
     <SamsungTizenSdkPackageVersion>7.0.100</SamsungTizenSdkPackageVersion>
     <!-- emsdk -->
-    <MicrosoftNETWorkloadEmscriptennet7Manifest70100PackageVersion>7.0.0-rtm.22504.3</MicrosoftNETWorkloadEmscriptennet7Manifest70100PackageVersion>
-    <MicrosoftNETWorkloadEmscriptenPackageVersion>$(MicrosoftNETWorkloadEmscriptennet7Manifest70100PackageVersion)</MicrosoftNETWorkloadEmscriptenPackageVersion>
+    <MicrosoftNETWorkloadEmscriptenPackageVersion>7.0.2</MicrosoftNETWorkloadEmscriptenPackageVersion>
     <!-- wasdk -->
     <MicrosoftWindowsAppSDKPackageVersion>1.2.230118.102</MicrosoftWindowsAppSDKPackageVersion>
     <MicrosoftWindowsSDKBuildToolsPackageVersion>10.0.22621.755</MicrosoftWindowsSDKBuildToolsPackageVersion>

--- a/eng/cake/dotnet.cake
+++ b/eng/cake/dotnet.cake
@@ -102,9 +102,14 @@ Task("dotnet-build")
     {
         RunMSBuildWithDotNet("./Microsoft.Maui.BuildTasks.slnf");
         if (IsRunningOnWindows())
+        {
             RunMSBuildWithDotNet("./Microsoft.Maui.sln", maxCpuCount: 1);
+        }
         else
-            RunMSBuildWithDotNet("./Microsoft.Maui-mac.slnf", maxCpuCount: 1);
+        {
+            // NOTE: intentionally omit maxCpuCount, to avoid an issue with the 7.0.100 .NET SDK
+            RunMSBuildWithDotNet("./Microsoft.Maui-mac.slnf");
+        }
     });
 
 Task("dotnet-samples")

--- a/src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj
+++ b/src/BlazorWebView/tests/MauiDeviceTests/MauiBlazorWebView.DeviceTests.csproj
@@ -7,6 +7,8 @@
     <RootNamespace>Microsoft.Maui.MauiBlazorWebView.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.MauiBlazorWebView.DeviceTests</AssemblyName>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Controls/samples/Controls.Sample.Profiling/Maui.Controls.Sample.Profiling.csproj
+++ b/src/Controls/samples/Controls.Sample.Profiling/Maui.Controls.Sample.Profiling.csproj
@@ -11,6 +11,8 @@
     <ApplicationVersion>1</ApplicationVersion>
     <IsPackable>false</IsPackable>
     <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Controls/samples/Controls.Sample.Sandbox/Maui.Controls.Sample.Sandbox.csproj
+++ b/src/Controls/samples/Controls.Sample.Sandbox/Maui.Controls.Sample.Sandbox.csproj
@@ -5,6 +5,8 @@
     <OutputType>Exe</OutputType>
     <SingleProject>true</SingleProject>
     <IsPackable>false</IsPackable>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj
+++ b/src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj
@@ -8,6 +8,8 @@
     <AssemblyName>Maui.Controls.Sample</AssemblyName>
     <IsPackable>false</IsPackable>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Controls/tests/DeviceTests/Controls.DeviceTests.csproj
+++ b/src/Controls/tests/DeviceTests/Controls.DeviceTests.csproj
@@ -7,6 +7,8 @@
     <RootNamespace>Microsoft.Maui.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.Controls.DeviceTests</AssemblyName>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Core/tests/DeviceTests/Core.DeviceTests.csproj
+++ b/src/Core/tests/DeviceTests/Core.DeviceTests.csproj
@@ -7,6 +7,8 @@
     <RootNamespace>Microsoft.Maui.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.Core.DeviceTests</AssemblyName>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/DotNet/DotNet.csproj
+++ b/src/DotNet/DotNet.csproj
@@ -4,14 +4,12 @@
     <IsPackable>false</IsPackable>
     <InstallWorkloadPacks Condition=" '$(InstallWorkloadPacks)' == '' ">true</InstallWorkloadPacks>
     <InternalAzureFeed>https://dotnetbuilds.blob.core.windows.net/internal</InternalAzureFeed>
-    <DotNetFeedUrl>https://dotnetbuilds.blob.core.windows.net/public</DotNetFeedUrl>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('windows'))">
     <DotNetInstallScriptUrl>https://dot.net/v1/dotnet-install.ps1</DotNetInstallScriptUrl>
     <DotNetInstallScriptName>dotnet-install.ps1</DotNetInstallScriptName>
     <DotNetInstallScriptPath>$(DotNetOutputPath)$(DotNetInstallScriptName)</DotNetInstallScriptPath>
     <DotNetInstallCommand>&amp; '$(DotNetInstallScriptPath)' -Version $(MicrosoftDotnetSdkInternalPackageVersion) -InstallDir '$(DotNetDirectory)' -Verbose</DotNetInstallCommand>
-    <DotNetInstallCommand Condition=" '$(PRIVATE_BUILD)' == 'false' ">$(DotNetInstallCommand) -AzureFeed $(DotNetFeedUrl)</DotNetInstallCommand>
     <DotNetInstallCommand Condition=" '$(PRIVATE_BUILD)' == 'true' ">$(DotNetInstallCommand) -AzureFeed $(InternalAzureFeed) -FeedCredential $env:DOTNET_TOKEN</DotNetInstallCommand> 
     <DotNetInstallCommand>powershell -ExecutionPolicy ByPass -NoProfile -Command &quot;$(DotNetInstallCommand)&quot;</DotNetInstallCommand>
   </PropertyGroup>
@@ -21,7 +19,6 @@
     <DotNetInstallScriptPath>$(DotNetOutputPath)$(DotNetInstallScriptName)</DotNetInstallScriptPath>
     <DotNetInstallCommand>bash '$(DotNetInstallScriptPath)' --version $(MicrosoftDotnetSdkInternalPackageVersion) --install-dir '$(DotNetDirectory)' --verbose</DotNetInstallCommand>
     <DotNetInstallCommand Condition=" '$(PRIVATE_BUILD)' == 'true' ">$(DotNetInstallCommand) --azure-feed $(InternalAzureFeed) --feed-credential $DOTNET_TOKEN</DotNetInstallCommand>
-    <DotNetInstallCommand Condition=" '$(PRIVATE_BUILD)' != 'true' ">$(DotNetInstallCommand) --azure-feed $(DotNetFeedUrl)</DotNetInstallCommand>
   </PropertyGroup>
 
   <PropertyGroup>
@@ -75,7 +72,7 @@
     </ItemGroup>
     <Copy SourceFiles="$(MauiRootDirectory)NuGet.config" DestinationFolder="$(DotNetTempDirectory)" />
     <Exec Command="dotnet nuget add source &quot;$(PackageOutputPath)&quot; --name artifacts --configfile &quot;$(DotNetTempDirectory)NuGet.config&quot;" />
-    <Exec Command="&quot;$(MSBuildExtensionsPath)../../dotnet&quot; workload install tizen maui --skip-manifest-update --verbosity diag --temp-dir &quot;$(DotNetTempDirectory)&quot; --configfile &quot;$(DotNetTempDirectory)NuGet.config&quot;" WorkingDirectory="$(MauiRootDirectory)" />
+    <Exec Command="&quot;$(MSBuildExtensionsPath)../../dotnet&quot; workload install tizen maui --skip-sign-check --skip-manifest-update --verbosity diag --temp-dir &quot;$(DotNetTempDirectory)&quot; --configfile &quot;$(DotNetTempDirectory)NuGet.config&quot;" WorkingDirectory="$(MauiRootDirectory)" />
 
   </Target>
 
@@ -168,7 +165,7 @@
       Inputs="$(_Inputs)"
       Outputs="$(DotNetPacksDirectory).stamp">
     <Exec
-        Command="&quot;$(DotNetToolPath)&quot; workload install %(_WorkloadIds.Identity) --skip-manifest-update --verbosity diag --temp-dir &quot;$(DotNetTempDirectory)&quot; --configfile &quot;$(MauiRootDirectory)NuGet.config&quot;"
+        Command="&quot;$(DotNetToolPath)&quot; workload install %(_WorkloadIds.Identity) --skip-sign-check --skip-manifest-update --verbosity diag --temp-dir &quot;$(DotNetTempDirectory)&quot; --configfile &quot;$(MauiRootDirectory)NuGet.config&quot;"
         WorkingDirectory="$(MauiRootDirectory)"
         EnvironmentVariables="DOTNET_MULTILEVEL_LOOKUP=0"
     />

--- a/src/Essentials/samples/Samples/Essentials.Sample.csproj
+++ b/src/Essentials/samples/Samples/Essentials.Sample.csproj
@@ -8,6 +8,8 @@
     <SingleProject>true</SingleProject>
     <IsPackable>false</IsPackable>
     <NoWarn>$(NoWarn),CA1416</NoWarn>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Essentials/test/DeviceTests/Essentials.DeviceTests.csproj
+++ b/src/Essentials/test/DeviceTests/Essentials.DeviceTests.csproj
@@ -6,6 +6,8 @@
     <SingleProject>true</SingleProject>
     <RootNamespace>Microsoft.Maui.Essentials.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.Essentials.DeviceTests</AssemblyName>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/Graphics/tests/DeviceTests/Graphics.DeviceTests.csproj
+++ b/src/Graphics/tests/DeviceTests/Graphics.DeviceTests.csproj
@@ -6,6 +6,8 @@
     <SingleProject>true</SingleProject>
     <RootNamespace>Microsoft.Maui.Graphics.DeviceTests</RootNamespace>
     <AssemblyName>Microsoft.Maui.Graphics.DeviceTests</AssemblyName>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

--- a/src/TestUtils/samples/DeviceTests.Sample/TestUtils.DeviceTests.Sample.csproj
+++ b/src/TestUtils/samples/DeviceTests.Sample/TestUtils.DeviceTests.Sample.csproj
@@ -6,6 +6,8 @@
     <SingleProject>true</SingleProject>
     <RootNamespace>Microsoft.Maui.TestUtils.DeviceTests.Sample</RootNamespace>
     <AssemblyName>Microsoft.Maui.TestUtils.DeviceTests.Sample</AssemblyName>
+    <!-- Disable multi-RID builds to workaround a parallel build issue -->
+    <RuntimeIdentifier Condition="$(TargetFramework.Contains('-maccatalyst'))">maccatalyst-x64</RuntimeIdentifier>
   </PropertyGroup>
 
   <PropertyGroup>

